### PR TITLE
Update SRG-OS-000420-GPOS-00186 for RHEL 9 STIG

### PR DESCRIPTION
**Pull request by jan-cerny**
#### Description:
* add fixtext and CCE
* explicitely set variable 

#### Rationale:
RHEL 9 STIG

### CODE REVIEW

File: controls/srg_gpos/SRG-OS-000420-GPOS-00186.yml [hunk 1]
```diff
@@ -8,4 +8,5 @@ controls:
         rules:
             - sysctl_net_ipv4_tcp_invalid_ratelimit
             - sysctl_net_ipv4_tcp_syncookies
+            - sysctl_net_ipv4_tcp_invalid_ratelimit_value=five_hundred
```
Comment (Mab879) [inline controls/srg_gpos/SRG-OS-000420-GPOS-00186.yml:11]: So in the current form of the script the value needs to be the value you want on the spreadsheet. Below is the current text.

```The command should output the following line:
/etc/sysctl.conf:net.ipv4.tcp_invalid_ratelimit = five_hundred```

**Review by Mab879 — CHANGES_REQUESTED** [review]

File: controls/srg_gpos/SRG-OS-000420-GPOS-00186.yml (hunk 1 above)
Comment (jan-cerny) [reply controls/srg_gpos/SRG-OS-000420-GPOS-00186.yml:11]: I'm sorry, I don't understand. Can you rephrase? What is the expected change here?

Comment (Mab879) [reply controls/srg_gpos/SRG-OS-000420-GPOS-00186.yml:11]: So the issue is #8612 so we can merge it as but, we have to fix the export script.

Comment (jan-cerny) [reply controls/srg_gpos/SRG-OS-000420-GPOS-00186.yml:11]: Aha, yes, now I understand. I run into the same problem 2 weeks ago in the PR https://github.com/ComplianceAsCode/content/pull/8555.

**Comment by jan-cerny** [comment]
The test error is `ERROR - No tests found matching the rule ID(s) 'sysctl_net_ipv4_tcp_invalid_ratelimit'` so I think it's fine.

File: controls/srg_gpos/SRG-OS-000420-GPOS-00186.yml (hunk 1 above)
Comment (Mab879) [reply controls/srg_gpos/SRG-OS-000420-GPOS-00186.yml:11]: #8612 has been fixed, so this should work once the branch is rebased.

**Review by Mab879 — APPROVED** [review]
